Uplift/Views/Dropdown/DropdownHeaderView.swift

```diff
@@ -13,32 +13,90 @@ protocol DropdownHeaderViewDelegate: class {
 }
 
 class DropdownHeaderView: UIView {
-
+    
+    var filtersAppliedCircle: UIView!
+    var selectedFiltersLabel: UILabel!
+    var titleLabel: UILabel!
+    
     private let arrowHeight: CGFloat = 9
     private let arrowImageView = UIImageView()
     private let arrowWidth: CGFloat = 5
     private var isArrowRotated = false
-
+    
     weak var delegate: DropdownHeaderViewDelegate?
+    
+    var filtersApplied: Bool = false {
+        didSet {
+            self.filtersAppliedCircle.layer.backgroundColor = filtersApplied ? UIColor.primaryYellow.cgColor : UIColor.primaryWhite.cgColor
```

Comment:
lets split across multiple lines since its kinda long:
```
self.filtersAppliedCircle.layer.backgroundColor = filtersApplied 
  ? UIColor.primaryYellow.cgColor 
  : UIColor.primaryWhite.cgColor
```